java_interop/protobuf-triple/go/go-client/conf/dubbogo.yml

```diff
@@ -0,0 +1,9 @@
+dubbo:
+  registries:
+    zk:
+      address: zookeeper://127.0.0.1:2181
+  Consumer:
+    services:
+      GreeterConsumer:
+        # interface is for registry
+        interface: org.apache.dubbo.demo.GreeterImpl
```

Comment:
Is this file still needed? I think we don't need this file anymore with the new API. Please confirm if we can delete this file.